engine/apps/schedules/models/shift_swap_request.py

```diff
@@ -166,6 +166,18 @@ def slack_channel_id(self) -> str | None:
         """
         return self.schedule.channel
 
+    @property
+    def beneficiary_slack_verbal(self) -> str:
+        return self.beneficiary.get_username_with_slack_verbal(True)
+
+    @property
+    def benefactor_slack_verbal(self) -> str | None:
```

Comment:
this one and the one above are not used anymore, right?

Reply:
correct 👍